### DISCUSSION TITLE
How do I hold down modifier keys in addition to a click?

### DISCUSSION THREAD

**Discussion by nchengyeeshen** [cymian/mouseless#37] [Q&A] [answered]
Does anybody have ideas on how I can hold down modifier keys like Cmd with a click action? When browsing the web, I tend to open links in a new tab by a combination of Cmd+click.

**Comment by cymian** [comment, marked as answer]
hi @nchengyeeshen, great question -- to do this, simply hold that key while hitting the action key (space or subcell).  so as long as commandLeft or commandRight isn't assigned to a "hold for" action or "execute mouse action", you should be able to use it for this 👍

this is an undocumented feature, but i should (and will) document it!

i've considered making cmd+click, alt+click, etc into their own actions, so you can execute them with one keypress instead of two, and also just for the freedom of using whatever key you'd like for these actions -- let me know if this is something you'd like to see!

**Comment by ognistik** [comment]
> i've considered making cmd+click, alt+click, etc into their own actions, so you can execute them with one keypress instead of two, and also just for the freedom of using whatever key you'd like for these actions -- let me know if this is something you'd like to see!

👆️ I'd love this

**Comment by michaelwills** [comment]
I know this is answered, but I am trying to get command + click to work. I don't have CommandLeft set to any action aside from CommandLeft + tab to show overlay, and for the speed increase. I am trying command left when selecting a subcell or space on the middle of the cell. Is there a step I am missing?

**Reply by cymian** [reply]
Ah yes, this is a bug I need to fix.  Wheel-related `hold for` commands (like speed increase) prevent modifier clicks, but there's no good reason they should.  Will fix for the 0.4 stable release.

I created #189 to track it

**Reply by michaelwills** [reply]
One of my "functional" tests right now is seeing if I can add a PR comment purely with `mouseless` 😅 .
